include/faabric/scheduler/MpiWorld.h

```diff
@@ -220,12 +224,11 @@ class MpiWorld
     std::string function;
 
     std::shared_ptr<state::StateKeyValue> stateKV;
-    std::unordered_map<int, std::string> rankHostMap;
+    std::vector<std::string> rankHostMap;
```

Comment:
I could use `std::array` instad of `std::vector`, but the changes would then be more involved, as I would have to change the way we initialise the `MpiWorld` (arrays need to know their size at compile time).